### ISSUE TITLE
Error encountered when generating AUC: Input contains NaN, infinity or a value too large for dtype('float32').

### ISSUE THREAD

**Issue by xiao-uofc** [jamesdolezal/slideflow#140]
## Description
During Validation metrics, the following error occurred -  
Error encountered when generating AUC: Input contains NaN, infinity or a value too large for dtype('float32').

## To Reproduce
1.
2.
3.

## Environment:

 - Slideflow Version (e.g., 1.0): 1.0.5
 - OS (e.g., Ubuntu): Ubuntu
 - How you installed PyTorch (`pip`, source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by jamesdolezal** [comment]
This happens when the model generates predictions that are NaN. There are many things that can cause this, but generally it is due to model divergence. There is an article on stackoverflow discussing this: https://stackoverflow.com/questions/40050397/deep-learning-nan-loss-reasons. The most likely cause is that the learning rate is too high, which caused the model to fail to converge, which will generate NaNs. A high dropout can also cause this.

Can you post the console output from the model training?

**Comment by xiao-uofc** [comment]
Dear James,

I think I understand the issue now. I just made the learning rate -> 10 times smaller and larger batch size. This should work now I hope. For the more detailed discussions we can chat on slack.

Regards, Xiao

**Comment by xiao-uofc** [comment]
Just to confirm - after adjusting the learning rate step size, the AUC part is working now. You are right - the loss couldn't converge was the issue. Thanks.
